input/fsh/DEF_RuleSets.fsh

```diff
@@ -51,6 +51,10 @@ RuleSet: LOINCCopyrightForVS
 * ^copyright = "This material contains content from LOINC (http://loinc.org). LOINC is copyright © 1995-2020, Regenstrief Institute, Inc. and the Logical Observation Identifiers Names and Codes (LOINC) Committee and is available at no cost under the license at http://loinc.org/license. LOINC® is a registered United States trademark of Regenstrief Institute, Inc"
 * ^experimental = false
 
+RuleSet: SNOINCCopyrightForVS
```

Comment:
Should there be different rulesets for the copyright notice for LOINC-based VS,  SNOMEDCT-based VS, and LOINC+SNOMEDCT-based valuesets?

Reply:
Well, yes, this is what is happening here... SNOINC = SNOMED + LOINC